src/euphorie/content/profilequestion.py

```diff
@@ -38,14 +39,14 @@ class IProfileQuestion(model.Schema, IRichDescription, IBasic):
     be skipped, or repeated multiple times.
     """
 
-    question = schema.TextLine(
+    question = HtmlText(
```

Comment:
:eyes: I'd have used `plone.app.textfield.RichText` instead and use Plone's portal_transform machinery with maybe a custom transform and also benefit from transform caching, etc.
But this is the way we do it in euphorie and we don't change that for this PR, alright.

Reply:
Yeah, I just want to be consistent for the moment. But maybe we can talk about switching if you see benefits there...